meta-mender-toradex-nxp/recipes-bsp/u-boot/u-boot-toradex-fsl-fw-utils_%.bbappend

```diff
@@ -0,0 +1 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
I would replace this with `require u-boot-mender-toradex-nxp.inc` so everything is in the same place.